lib/project/pro_motion/data_table.rb

```diff
@@ -0,0 +1,187 @@
+module ProMotion
+  module DataTable
+    include ProMotion::Styling
+    include ProMotion::Table::Utils
+
+    def table_view
+      self.view
+    end
+
+    def screen_setup
+      set_up_reload_notification
+      set_up_fetch_controller
+
+      # TODO - implement dynamic row height
+      # set_up_row_height
+    end
+
+    def set_up_reload_notification
+      NSNotificationCenter.defaultCenter.addObserver(self,
+        selector: "update_table_data:",
+        name: NSManagedObjectContextDidSaveNotification,
+        object: nil)
+    end
+
```

Comment:
It looks like you are listening for a context save notification generally.  That's probably not what you want, especially in connection with the merge in `update_table_data`.  You really want to know when the specific context that's connected with your fetch controller saves, not when any context anywhere saves.